README.md

```diff
@@ -78,27 +108,78 @@ const quoteResponse = await cowSdk.cowApi.getQuote({
 })
 
 const { sellToken, buyToken, validTo, buyAmount, sellAmount, receiver, feeAmount } = quoteResponse.quote
+```
+
+### STEP 2: Sign the order
+Once you know the price and fee, we can create the order and sign it:
+
+  * Technically the order is just a signed message with your intent to trade, and contains your `Limit Price` and `Fee`.
+  * As explained before, you can choose your `Limit Price`, but some general approach is to take the current Market Price
+  and apply some slippage tolerance to it. `Received Amount = Expected Amount * (1 - Slippage Tolerance)`
+  * The SDK will provide an easy way to sign orders given the raw data
+
+
+```js
+const { sellToken, buyToken, validTo, buyAmount, sellAmount, receiver, feeAmount } = quoteResponse.quote
+
+// Prepare the RAW order
 const order = {
-  kind: OrderKind.SELL,
-  partiallyFillable: false, // Allow partial executions of an order (true would be for a "Fill or Kill" order, which is not yet supported but will be added soon)
+  kind: OrderKind.SELL, // SELL / BUY  
+  receiver, // Your account or any other
   sellToken,
   buyToken,
+
+  partiallyFillable: false, // (true would be for a "Fill or Kill" order, "false" for allowing "Partial execution" which is not supported yet)
+
+  // Deadline
   validTo,
-  buyAmount,
+
+  // Limit Price
+  //    You can apply some slippage tolerance here to make sure the trade is executed. 
+  //    CoW protocol protects from MEV, so it can work with higher slippages
   sellAmount,
-  receiver,
+  buyAmount, 
+
+  // Use the fee you received from the API
   feeAmount,
+
+  // The appData allows you to attach arbitrary information (meta-data) to the order. Its explained in their own section. For now, you can use this 0x0 value
+  appData: '0x0000000000000000000000000000000000000000000000000000000000000000'
 }
 
-// 3. Sign the order using your wallet
+// Sign the order
 const signedOrder = await cowSdk.signOrder(order)
+```
+
+At this point, you have a signed order. So next step will be to post it to the API so it's considered by the solvers and executed.
+
+  
+## STEP 3: **Post the signed order to the API**: 
+Once you have a signed order, last step is to send it to the API.
+  * The API will accept the order if its correctly signed, the deadline is correct, and the fee is enough to settle it
+  *  Once accepted, the order will be `OPEN` until the specified `validTo` date (expiration)
+  * The possible outcomes once is accepted is:
+      * The order is `EXECUTED`: you will pay the signed fee, and get at least the `buyAmount` tokens you specified, although you will probably get more! (you will probably get a so-called **Surplus**).
```

Comment:
Should we make clear this is in regards to a sell order?
Should we describe how it would work for buy orders?